src/test_l1_dsHdmiIn.c

```diff
@@ -2008,22 +2018,20 @@ void test_l1_dsHdmiIn_negative_dsGetAllmStatus(void) {
 void test_l1_dsHdmiIn_positive_dsGetSupportedGameFeaturesList(void) {
     gTestID = 43;
     UT_LOG("\n In %s [%02d%03d]\n", __FUNCTION__, gTestGroup, gTestID);
+
```

Comment:
Line no : 2013 Modify the descriptions and steps as we are now comparing with the profile file.

Reply:
Done